### PR TITLE
CAPSTONE (AMS)

### PR DESCRIPTION
**Pull request by Jessekay**
### **THROUGH THIS MILESTONE I IMPLEMENTED THE FOLLOWING:**

- I personalized the content of my page. Choose a topic that is about African movie festivals.
- I followed all design colors
- I made the logo link to the homepage
- When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
- Featured speakers were created in javascript
- Lastly, I recorded myself describing my project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,155 @@
...
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```
Comment (alexander16108) [inline README.md:154]: Hi, could you delete this section it is irrelevant to the details of your readme file. 😄

File: index.html [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+              <footer class='footer container-2'>
+                <div class=' footer-details '>
+                  <div class="common">
+                    <img src="./images/cc-logo2.png" alt="logo">
+                  </div>
+                  <div class="copyright">
+                    <p>2023 African movies. Some Rights Reserved.</p>
+                  </div>
+                </div>
+              </footer>
```
Comment (alexander16108) [inline index.html:205]: - in the mobile version as seen below your partner and footer sections should both be hidden and should only appear in the desktop version.

your design
image
 
cindy's design
image

- in your mobile version there is no `MORE ` button to show the full list of speakers or  `LESS` button to only see a few speakers as seen below

your version
image

cindy's version
image

File: about.html [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+            <img src="./images/cc.logo.white.svg" class='mobile'   alt="logo">
+            <img src="./images/cc.logo.white.svg"  class='desktop-only' alt="logo">
+            
+          </div>
+          <div class="abt-copyright">
+            <p>2023 African Movies. Some Rights Reserved.</p>
+          </div>
+        </div>
+      </footer>
+
```
Comment (alexander16108) [inline about.html:153]: 👋🏾  in the footer section of your about page, you ought to have both the partners section and the footer together in the mobile version but you only have the footer. please add this as it is required that your design remains identical to the template design

your design mobile version:
image


cindy's design
image

File: README.md [hunk 4]
```diff
@@ -1 +1,155 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="Cindy Shin in Behance"></a>
+
+
+
+I would like to thank Cindy Shin in Behance for the amazing UX
```
Comment (alexander16108) [inline README.md:141]: it's required to give proper appreciation to Cindy for the template design with links to her profile and design 👍🏾 
*** copy the text from below and put it in the acknowledgement section of your readme file.

> I would like to give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/)

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @Jessekay , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ichala — APPROVED** [review]
Hi Student  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**